app/components/layout/TopBarLayout.scss

```diff
@@ -1,6 +1,7 @@
 $topBarClassicHeight: 64px;
 $topBarHeight: 58px;
```

Comment:
It's repeating at 3-4, it will be fixed in next PR (fixing now will need re-base in merge pending PRs)
https://app.clubhouse.io/emurgo/story/1026/fix-repeating-topbarclassicheight-and-topbarheight-in-scss